File: browservn/src/androidTest/java/org/sqlunet/browser/vn/ColorsNightTest.java
/*
 * Copyright (c) 2023. Bernard Bou
 */

package org.sqlunet.browser.vn;

import org.junit.runner.RunWith;

import androidx.appcompat.app.AppCompatDelegate;
import androidx.test.ext.junit.runners.AndroidJUnit4;
import androidx.test.filters.LargeTest;

@RunWith(AndroidJUnit4.class)
@LargeTest
public class ColorsNightTest extends AbstractColors
{
	@Override
	int getMode()
	{
		return AppCompatDelegate.MODE_NIGHT_YES;
	}
}

File: browserfn/src/androidTest/java/org/sqlunet/browser/fn/ColorsNightTest.kt
/*
 * Copyright (c) 2023. Bernard Bou
 */
package org.sqlunet.browser.fn

import androidx.appcompat.app.AppCompatDelegate
import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import org.junit.runner.RunWith

@RunWith(AndroidJUnit4::class)
@LargeTest
class ColorsNightTest : AbstractColors() {

    override val mode: Int
        get() = AppCompatDelegate.MODE_NIGHT_YES
}
